lib/svg-sprite/shape.js

```diff
@@ -730,17 +731,17 @@ SVGShape.prototype.setNamespace = function(ns) {
         if (namespaceClassnames) {
             // Build a class name substitution table (and alter the elements' class names accordingly)
             substClassnames = {};
-            select('//*[@class]', this.dom).forEach(elem => {
+            for (const elem of select('//*[@class]', this.dom)) {
                 const elemClassnames = [];
-                elem.getAttribute('class').split(' ')
-                    .filter(classname => classname.trim())
-                    .forEach(classname => {
-                        const substClassname = ns + classname;
-                        substClassnames[`.${classname}`] = substClassname;
-                        elemClassnames.push(substClassname);
-                    });
+                for (const classname of elem.getAttribute('class').split(' ')
```

Comment:
We could/should move these kind of long expressions to variables before the loop